data/series.txt

```diff
@@ -17,3 +17,22 @@ category "series"
 	"Human: Plasma" 120
 	"Human: Atomic" 130
 	"Human: Afterburners" 140
+
+mission "relocate"
+	repeat
+	on offer
+		relocate
+			"flagship only"
+			location
+				planet "Earth"
+		conversation
+			`You're going to Detroit.`
+			choice
+				`	"NOOOOOOOOOOOOOOOOOOOOOO."`
+					goto test
+				`	"I refuse"`
+			`	"Too bad."`
+				decline
+			label test				
+			`	YES`
+			 accept
```

Comment:
~~are we really going to relocate ppl to Detroit in vanilla?~~

Reply:
Yes, this PR gets only shipped with this mission

Reply:
~I might back down from this point~